docs/proposals/2019-03-18-Links-List.md

```diff
@@ -0,0 +1,113 @@
+## Problem
+
+Organizations have requested a way to include sidebars, related content boxes, or other recirculation links within the body of an article. A curated solution -- where users have reasonable control over the presentation and selection of the items on an individual -- is highly-desirable.
+
+Existing solutions are either too generic (`element_group`, `story`) to interpret well in a UX, or too limited (`related_content`, `promo_items`) to be useful in this capacity, or cannot be meaningfully extended in a non-breaking way.
+
+
+## Proposal
+
+Add a new content element `link_list` that contains a narrowly-targeted set of functionality around inline recirculation boxes. Applicable to stories only.
+
+A links_list may have:
+* A plaintext title
+* A subtype (selected from an enumerated-list configured on a per-organization basis)
+* A sequence of up to 10 and at least one `link_list_item`s, each of which has:
+    * A subtype
+    * A plaintext title (required)
+    * A url (required)
+    * A description (ANS `text` element)
+    * An image (ANS `image` or `reference` to an image)
+
+
+## Example
+
+```json
+{
+  "type": "story",
+  "version": "0.10.x",
+
+  "headlines": {
+    "basic": "Man Bites Dog"
+  },
+
+  "content_elements": [
+    {
+      "type": "link_list",
+
+      "subtype": "sidebar",
+      "title": "More 'Biting' Stories",
```

Comment:
I don't think the title for the type or the items should be required.  

Use cases I see for this include: 
- support the use of a "read more" list. 
In this case, the set of links has a title, "Read More:", but the individual links don't.  See https://www.washingtonpost.com/opinions/2019/03/25/trump-won-with-illicit-help-he-abused-his-power-his-ag-is-blocking-full-reckoning/?utm_term=.cfcdb102bdb2  

- replace the use of content element type "interstitial_link".  
In this case, in interstitial link is really a set of 1, with no headlines.

Reply:
What would the link text be if not the title?

Reply:
I think maybe what @StephanieDClark is getting at here is there are examples where there is supporting text for a link that is not the link itself? I know that historically this has been available (especially with interstitial links) but was not able to find a recent example.

Reply:
From the journalist's perspective, the interstitial_link content element requires fewer steps than using a links list with a _interstitial_link_ subtype and no headline